dbt/include/athena/macros/adapters/relation.sql

```diff
@@ -36,6 +36,14 @@
   {%- endcall %}
 {%- endmacro %}
 
+{% macro set_table_relation_schema(relation, schema) %}
+  {%- if temp_schema is not none -%}
+    {%- set relation = relation.incorporate(path={"schema": schema}) -%}
+    {%- do create_schema(relation) -%}
```

Comment:
I'm not 100% convinced by this. pretty_much create_schema will be called by every model where we setup the temp_schema. It will be good too find a a better way to handle that.